### ISSUE TITLE
Galai library doesn't seem to work

### ISSUE THREAD

**Issue by saptarshi059** [paperswithcode/galai#55]
Hi, 

I'm trying to use the galai quickstart prompt. This is what I'm running: 

```
import galai as gal

model = gal.load_model("standard")
model.generate("Scaled dot product attention:\n\n\\[")
# Scaled dot product attention:\n\n\\[ \\displaystyle\\text{Attention}(Q,K,V)=\\text{softmax}(\\frac{QK^{T}}{\\sqrt{d_{k}}}%\n)V \\]

```
However, I'm getting `TypeError: 'type' object is not subscriptable`

**Comment by mkardas** [comment]
Hi @saptarshi059, I cannot reproduce this. Can you try with galai==1.1.1? If it's still broken can you paste in the full stack trace?

**Comment by mkardas** [comment]
Based on the stack trace from #59 the issue should be fixed with `galai==1.1.2`. It didn't fail for me as I was testing it on python 3.9.

**Comment by saptarshi059** [comment]
@mkardas Thank you for the solution from #59 ... I upgraded accelerate and it now works!
